runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -128,6 +140,11 @@
 	private int lastAnimation;
 	private boolean loggedInRace;
 	private boolean widgetHiddenChangedOnPvpWorld;
+	private ElapsedTimer tzhaarTimer;
+	private Instant tzhaarStartTime;
+	private Instant tzhaarLastTime;
+	private Boolean tzhaarStarted = false;
```

Comment:
As was discussed on Discord, why is this needed in addition to `tzhaarStartTime`? Is this always used as a `tzhaarStartTime != null` check?